nequip/scripts/train.py

```diff
@@ -71,7 +72,7 @@ def main(args=None, running_as_script: bool = True):
     if running_as_script:
         set_up_script_logger(config.get("log", None), config.verbose)
 
-    found_restart_file = isdir(f"{config.root}/{config.run_name}")
+    found_restart_file = exists(f"{config.root}/{config.run_name}/trainer.pth")
```

Comment:
In principle I'm fine with this change, but I want to make sure that it can't lead to logically inconsistent behavior---do we need to add something where it clears the entire rundir if `trainer.pth` isn't found, so that it is truly a fresh start in the case where the directory exists but `trainer.pth` does not?

Reply:
So checking through the code and testing with `nequip-train`, if the directory exists but `trainer.pth` does not, then the code does act like a fresh start if `append` is `True`, but otherwise will crash if not because it detects that the log files are already present (https://github.com/mir-group/nequip/blob/3db69640fabb684db1e13ec56826c3cb6586fafc/nequip/utils/output.py#L87).
I think the best thing would be to clear the folder in these cases for full consistency. I think it might be good to add a warning when this is happening as well? Will add these

Reply:
[94979e8](https://github.com/mir-group/nequip/pull/438/commits/94979e86af9377b4eeead6665bc230441abd2542)